services/QuillLMS/client/app/bundles/Teacher/components/salesForm/lowerFormFields.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { Input } from '../../../Shared';
+import { SCHOOL_PREMIUM_ESTIMATE, TEACHER_PREMIUM_ESTIMATE, STUDENT_PREMIUM_ESTIMATE, COMMENTS } from '../../../../constants/salesForm';
+
+export const LowerFormFields = ({ errors, handleUpdateField, schoolPremimumEstimate, teacherPremimumEstimate, studentPremimumEstimate, comments }) => {
+  return(
+    <React.Fragment>
+      <Input
+        className="form-input estimate"
+        error={errors[SCHOOL_PREMIUM_ESTIMATE]}
+        handleChange={handleUpdateField}
+        id={SCHOOL_PREMIUM_ESTIMATE}
+        label={SCHOOL_PREMIUM_ESTIMATE}
+        placeholder=""
+        value={schoolPremimumEstimate}
+      />
+      <Input
+        className="form-input estimate"
+        error={errors[TEACHER_PREMIUM_ESTIMATE]}
+        handleChange={handleUpdateField}
+        id={TEACHER_PREMIUM_ESTIMATE}
+        label={TEACHER_PREMIUM_ESTIMATE}
+        placeholder=""
+        value={teacherPremimumEstimate}
+      />
+      <Input
+        className="form-input estimate"
+        error={errors[STUDENT_PREMIUM_ESTIMATE]}
+        handleChange={handleUpdateField}
+        id={STUDENT_PREMIUM_ESTIMATE}
+        label={STUDENT_PREMIUM_ESTIMATE}
+        placeholder=""
+        value={studentPremimumEstimate}
+      />
+      <div className="comments-container">
+        <label htmlFor={COMMENTS} id="comments-label">Comments (optional)</label>
+        <textarea
```

Comment:
I think this component: services/QuillLMS/client/app/bundles/Shared/components/shared/textArea.tsx might serve your purpose and already be styled to match the rest of the inputs here.

Reply:
ahh totally forgot about this one! I thought we only had the `TextEditor` component. I'll update